pyproject.toml

```diff
@@ -1,17 +1,36 @@
-[[source]]
-url = "https://pypi.org/simple"
-verify_ssl = true
-name = "pypi"
+[tool.poetry]
+name = "conduit"
+version = "0.1.0"
+description = "Pyramid and OpenAPI3 based RealWorld.io example."
+authors = ["niteo.co <info@niteo.co>"]
+license = "MIT"
+classifiers = [
+	"Programming Language :: Python",
+	"Programming Language :: Python :: 3.8",
+	"Framework :: Pyramid",
+	"Topic :: Internet :: WWW/HTTP",
+	"Topic :: Internet :: WWW/HTTP :: WSGI :: Application",
+	"License :: MIT",
+]
+homepage = "http://github.com/niteoweb/pyramid-realworld-example-app"
+keywords = ["pyramid", "openapi", "realworld"]
+packages = [
+	{ include = "conduit", from = "src" },
+]
```

Comment:
@zupo Looks like we don't need setup.py.

Reply:
Sure, I'm all for removing files if they are not needed.

Reply:
Yeah, looks good! Can you please do the same for kai-app repo?